README.md

```diff
@@ -76,106 +84,40 @@ A `Result` models the result of a computation chain in a functional way. In othe
 
 The point here is the same as for `Optional` (safety), but with a focus on readability. Returning a `Result` from a function signals that your function may fail and gives you details about the failure when it occurs. In certain contexts, you can also see it as an interruptible `Promise` chain. If you never liked `async`/`await` because of the verbosity of error handling using `try`/`catch`, then you'll love `Result`. A nice advantage is that it makes it easy to be specific about errors without writing tons of try catch blocks, which makes your error-handling code more testable.
 
-
-
-### Recipes
-
-
-
-#### `Optional` recipes
-
-
-
-* [⚓](#optional-construction) Constructing an `Optional`.
```

Comment:
I think those are very valuable. They act as some kind of cheatsheet that's very useful if you don't want to read 5 pages of text to find what you need. Not sure why they've been removed?

Reply:
Wasn't clear to me what they were or how to use them.
I can re-introduce them, but I'll change it into a TOC that also mentions the method's name.

Reply:
I would love that - thanks for the hard work!